docker/flags.go

```diff
@@ -28,6 +28,13 @@ func getHomeDir() string {
 	return os.Getenv("HOME")
 }
 
+func getDaemonConfDir() string {
+	if runtime.GOOS == "windows" {
+		return filepath.Join(os.Getenv("USERPROFILE"), ".docker")
```

Comment:
we should remember to change this when the daemon will run on windows, otherwise we will get the same issue.